lib/reek/ast/object_refs.rb

```diff
@@ -3,25 +3,48 @@
 module Reek
   # Represents functionality related to an Abstract Syntax Tree.
   module AST
-    # Responsible for holding one specific object reference.
-    ObjectRef = Struct.new(:name, :line)
```

Comment:
One constant less in our name system! Everybody wins!
Apart from that, this thing was super confusing since the `name` was completely redundant.